.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
We should rarely check in comments. The only case i can think of is TODOs and if you are stuck doing something hacky and want teammates to know why it's hacky cause it's not instantly apparent from reading code.